spec/arbre/unit/component_spec.rb

```diff
@@ -13,25 +13,25 @@ def build
 end
 
 describe Arbre::Component do
-  let(:assigns) { {} }
-  let(:helpers) { nil }
-  let(:component_class) { MockComponent }
-  let(:component) { component_class.new }
+  let(:assigns)         { {}                    }
+  let(:helpers)         { nil                   }
+  let(:component_class) { MockComponent         }
+  let(:component)       { component_class.new   }
```

Comment:
Whichever rule this is, it should be disabled. Once you add the next longer line then all of these have to be changed to match. I've run into that on the first try several times that this just adds no value. Enforcing the spacing between elements is more than sufficient.

There are multiple instances of this here so I won't comment on each one but all should be addressed.

Reply:
> Whichever rule this is, it should be disabled.

I think there are some linters missing in the `Style` scope. I can go for that first

Reply:
Identified (and disabled):

```
RSpec/AlignLeftLetBrace:
  Enabled: false

RSpec/AlignRightLetBrace:
  Enabled: false
```